doc/rados/operations/placement-groups.rst

```diff
@@ -53,7 +53,7 @@ For example, to enable autoscaling on pool ``foo``, run the following command:
 
    ceph osd pool set foo pg_autoscale_mode on
 
-There is also a ``pg_autoscale_mode`` setting for any pools that are created
+There is also a central config default ``pg_autoscale_mode`` option pools that are created
```

Comment:
This is not quite a sentence. Stripped down, this says "There is also an option pools that are created after setup", which is missing "for any" before "pools".

Reply:
pushing